src/decisionengine/framework/util/maybe_call.py

```diff
@@ -0,0 +1,39 @@
+# SPDX-FileCopyrightText: 2017 Fermi Research Alliance, LLC
```

Comment:
I wonder if this is more pythonic...

```python
from contextlib import suppress

with suppress(AttributeError):
    logger.debug(...)
```

If the shorter code is preferred, we could probably replace the class creation with the suppress.

Reply:
Are you suggesting that the `maybe(logger)` facility could be implemented using `contextlib.suppress`?  Or are you suggesting that the `maybe(logger)` facility could be removed, and places like `maybe(logger).debug(...)`  would be replaced with what you suggest?

Reply:
Okay, let me think about that this weekend.  Perhaps I've created a utility that is less helpful than I realized.

Reply:
@jcpunk, I've decided to remove the `maybe()` functionality altogether--it was overkill.  Will update PR